### DISCUSSION TITLE
IaaS vs. PaaS vs. SaaS

### DISCUSSION THREAD

**Discussion by starshb** [starshb/pharmacy_Find#5] [Q&A] [answered]
IaaS vs. PaaS vs. SaaS 공부 해올것

**Comment by tjdud22** [comment]
Infrastructure as a Service(IaaS, 아이아스, 이에스)서비스로 제공되는 인프라스트럭처입니다. 개발사에 제공되는 물리적 자원을 가상화합니다.
Platform as a Service(PaaS, 파스)서비스로 제공되는 플랫폼입니다. 개발사에 제공되는 플랫폼을 가상화합니다.
Software as a Service(SaaS, 사스)서비스로 제공되는 소프트웨어입니다. 고객에게 제공되는 소프트웨어를 가상화합니다.

**Comment by tjdud22** [comment, marked as answer]
IaaS : 서비스로 제공하는 인프라스트럭처
클라우드 인프라스트럭처 서비스는 확장성이 높고 자동화된 컴퓨팅 리소스를 가상화하여 제공하는 것입니다.

PaaS : 서비스로 제공하는 플랫폼
클라우드 플랫폼 서비스는 주로 응용 프로그램을 개발할 때 필요한 플랫폼을 제공하는 것입니다.

SaaS : 서비스로 제공하는 소프트웨어
클라우드 애플리케이션(소프트웨어) 서비스는 사용자에게 제공되는 소프트웨어를 가상화하여 제공하는 것입니다. SaaS는 타사 공급 업체가 관리하는 사용자에게 응용 프로그램을 제공하기 위해 인터넷을 사용합니다. 대부분의 SaaS 애플리케이션 웹 브라우저를 통해 직접 실행되므로 클라이언트 측에서 다운로드나 설치가 필요하지 않습니다.
